doc/todo/NEWS.md

```diff
@@ -48,52 +48,63 @@ You can also read the news [on our website](https://www.libelektra.org/news/0.8.
 ### <<HIGHLIGHT2>>
 
 
-## Other New Features
```

Comment:
I think it had some value to have a list of important new features (for advertisement) because sometimes nobody wanted to write a text for the highlight. Then we still could put the feature here.

Reply:
You can always add this text if you want highlight something for a specific release, but basically I think we should avoid categories, that overlap with other categories. In my opinion it should be easy to decide where to search for a certain item of interest.

Reply:
Yes, that is a good point.

My idea was to have the maximum number of categories in the template. (Also to avoid merge conflicts.) And shortly before the release we can merge categories that are hardly used.

The short sentences might help to distinguish where to put something.

Reply:
> My idea was to have the maximum number of categories in the template. (Also to avoid merge conflicts.) And shortly before the release we can merge categories that are hardly used.

Adding many (non-overlapping) sections for topics is a good idea, but adding sections that are too general does not help in removing merge conflicts. Categories like “Other New Features” or “Notes for Elektra's Developers” apply to more or less every Elektra update. For example, it is not really clear if an new feature of a plugin should be part of “Other New Features” or “Plugins”.

> The short sentences might help to distinguish where to put something.

Could you provide an example for a news item where it is not clear where to put it? I only had problems with the Docker and Jenkins related news items, since I do not know much about both  of those topics.